practice_problems_2/wordsFinding/main.php

```diff
@@ -0,0 +1,20 @@
+<?php
+// This function returns all words that start with the given starting letter
+function start_letter ($input, $starting_letter) {
```

Comment:
space should not be added after function name

Reply:
fixed